src/main/java/com/arpnetworking/clusteraggregator/aggregation/AggMessageExtractor.java

```diff
@@ -65,8 +78,8 @@ public String entityId(final Object message) {
             for (final Map.Entry<String, String> dimensionEntry : sortedDimensionsMap.entrySet()) {
                 final String k = dimensionEntry.getKey();
                 if (!(k.equals(CombinedMetricData.CLUSTER_KEY)
-                        || k.equals(CombinedMetricData.HOST_KEY)
-                        || k.equals(CombinedMetricData.SERVICE_KEY))) {
+                        || k.equals(CombinedMetricData.SERVICE_KEY)
+                        || _reaggregationDimensions.contains(k))) {
```

Comment:
Can we just put host, service, and cluster in _reaggregationDimensions?

Reply:
Host is in there by default. Cluster and Service still have some special meaning unfortunately.

Reply:
[Ref](https://github.com/ArpNetworking/metrics-cluster-aggregator/blob/master/config/config.conf#L52)